recognition/conversion.py

```diff
@@ -19,7 +19,7 @@ def __init__(
         parallelize=False,
         encoding="utf-8",
         fill_empty_segments=False,
-        best_path_algo=None,  # defaults to dijkstra, which does not work on graphs with negative edge-weights
+        best_path_algo="bellman-ford",  # default is dijkstra, which does not work on graphs with negative edge-weights
```

Comment:
please remove the comment, it is not relevant anymore ("default" might confuse the reader as the default in this job is now bellman-ford)